src/strategies/views/room-view.test.ts

```
@@ -0,0 +1,9 @@
+import { describe, expect, test } from 'vitest';
+import { RoomViewStrategy } from './room-view';
+
+describe('RoomViewStrategy.generate', () => {
+  test('returns view with empty lists', async () => {
+    const view = await RoomViewStrategy.generate({}, {} as any);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)